src/wasm.h

```diff
@@ -90,6 +90,9 @@ struct Name : public cashew::IString {
 
 // An index in a wasm module
 typedef uint32_t Index;
```

Comment:
In wasm-binary.h, in mapLocals() line 586, I added a toIndex(). Looks like it will always assert. However, travis passes so I'm thinking the code is never executed. I can't tell if curr is suppose to be equal to size() correctly, or if curr is off by 1, or it's just a work in progress. Can you have a look?